zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/package.kt

```diff
@@ -0,0 +1,26 @@
+package app.cash.zipline.kotlin
+
+import org.jetbrains.kotlin.name.CallableId
+import org.jetbrains.kotlin.name.ClassId
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.name.Name
+
+/** Represents a package name without an associated class. */
+@JvmInline
+value class FqPackageName(val fqName: FqName)
+
+fun FqPackageName(name: String): FqPackageName {
+  return FqPackageName(FqName(name))
+}
+
+fun FqPackageName.classId(name: String): ClassId {
```

Comment:
Nice API. I was excited to switch my compiler plugin when I saw it above, but now I'm sad because it's not standard.